### PR TITLE
Populate user ID for locally imported scores

### LINKED ISSUE #19012: Profile Picture of the user is not displayed when opening a replay.
### Type

Cosmetic

### Bug description

Below is replicated by downloading any replay and opening said replay into lazer.

### Screenshots or videos

osu_2022-07-06_13-19-22
https://osu.ppy.sh/scores/osu/4209619408 (Link to play)

### Version

2022.704.0-lazer

### Logs

[updater.log](https://github.com/ppy/osu/files/9051212/updater.log)
[network.log](https://github.com/ppy/osu/files/9051213/network.log)
[runtime.log](https://github.com/ppy/osu/files/9051214/runtime.log)
[performance.log](https://github.com/ppy/osu/files/9051215/performance.log)
[database.log](https://github.com/ppy/osu/files/9051216/database.log)

### PR DESCRIPTION
**Pull request by andrewhong04**
- [x] Depends on #19074 (for https://github.com/peppy/osu/commit/8b6665cb5b211c06e24f9d3e9bf210480bc08760)

Fixes: https://github.com/ppy/osu/issues/19012
Here's the replay I used: https://cdn.discordapp.com/attachments/935341632423854152/994808046431780944/Piixers_playing_Aimer_-_Black_Birdaads.osr

### CODE REVIEW

File: osu.Game/OsuGame.cs
Comment (peppy) [inline osu.Game/OsuGame.cs:553]: This seems like a very weird place to do this lookup. Is there a reason it's not done as part of the import process instead?

Comment (frenzibyte) [reply osu.Game/OsuGame.cs:553]: Already mentioned that in thread as well, `ScoreImporter.PostImport` should be a good place for such logic.

Comment (novialriptide) [reply osu.Game/OsuGame.cs:553]: > This seems like a very weird place to do this lookup. Is there a reason it's not done as part of the import process instead?

I couldn't find an effective way to import the API to `PostImport`, which is why I'm sending the api request here.

File: osu.Game/Database/RealmArchiveModelImporter.cs
Comment (novialriptide) [inline osu.Game/Database/RealmArchiveModelImporter.cs:341]: This was introduced in https://github.com/peppy/osu/commit/8b6665cb5b211c06e24f9d3e9bf210480bc08760

**Comment by frenzibyte** [comment]
Have pushed minor changes to appease CI and remove unnecessary debug logging. Seems fine as a first step, but will wait for a second approval.

**Review by frenzibyte — APPROVED** [review]

**Comment by peppy** [comment]
Seems to work fine for now.

**Review by peppy — APPROVED** [review]

**Pull request merged**
